src/contributing/usage-statistics.md

```diff
@@ -0,0 +1,24 @@
+# Usage Statistics
+
+If you would like to contribute usage reports, the
+[PopCorn](https://github.com/the-maldridge/popcorn) program reports installation
+statistics back to the Void project. PopCorn only reports which packages are
+installed, and the output of `xuname` for statistics on which platforms are
+being used.
```

Comment:
`This does not report services installed or enabled, or anything other than installed packages. In this fashion, it avoids almost all personally identifiable information.`

Reply:
I brought some of that language in, but do not understand the difference between installed packages and installed services.